src/mlpack/methods/range_search/rs_model_impl.hpp

```diff
@@ -59,6 +59,23 @@ inline RSModel::RSModel(RSModel&& other) :
   other.rSearch = decltype(other.rSearch)();
 }
 
+inline bool RSModel::operator==(RSModel other)
+{
+  if ( (this->treeType == other.treeType) && (this->leafSize == other.leafSize)
+                                     && (this->randomBasis == other.randomBasis)
+                                     && (this->rSearch == other.rSearch)
+                                     && (this->q.n_cols ==other.q.n_cols)
+                                     && (this->q.n_rows == other.q.n_rows) )
+  {
+    for (size_t i = 0; i < this->q.n_elem ; i++)
+      if (this->q[i] != other.q[i] )
```

Comment:
Weird spacing here too, please change to:

```
for (size_t i = 0; i < this->q.n_elem; i++)
  if (this->q[i] != other.q[i])
...
```

Also, the `this->` is not really necessary.